Makefile

```diff
@@ -97,7 +97,7 @@ ifneq ($(GIT_BASEDIR),)
 endif
 
 test: generate
-	ginkgo -skipPackage test/e2e -r .
+	ginkgo -skipPackage test/e2e/dcos,test/e2e/kubernetes -r .
```

Comment:
Added unit tests in the test/e2e package, we don't want to skip all of those anymore